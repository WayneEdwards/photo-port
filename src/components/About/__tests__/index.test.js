import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import About from '..';

const { asFragment } = render(<About />);

afterEach(cleanup);
describe('About component', () => {
    // renders About test
    // First Test
    it('renders', () => {
        render(<About />);
      });

    // Second Test 
    it('matches snapshot DOM node structure', () => {
        // render About
      })
  });

  expect(asFragment()).toMatchSnapshot();


