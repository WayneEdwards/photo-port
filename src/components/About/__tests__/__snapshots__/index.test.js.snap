// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<DocumentFragment>
  <section>
    <h1
      id="about"
    >
      Who am I
    </h1>
    <img
      alt="cover"
      class="my-2"
      src="cover-image.jpg"
      style="width: 100%;"
    />
  </section>
</DocumentFragment>
`;
